### PR TITLE
validate openAi api key and openAi error handling

### PR DESCRIPTION
**Pull request by vimal-oneway**
errorMesgChatDockX
Screenshot 2023-04-18 111141

### CODE REVIEW

File: src/utils/ValidateApiKey.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Configuration, OpenAIApi } from "openai";
```
Comment (Royal-lobster) [inline src/utils/ValidateApiKey.ts:1]: I believe we can verify without `openai`. with axios and catch.

**Review by Royal-lobster — CHANGES_REQUESTED** [review]
Looking great !! added some comments.

File: package-lock.json
[the remark this answers is not in the document]
Comment (vimal-oneway) [reply package-lock.json:1]: okay:)

File: src/utils/ValidateApiKey.ts (hunk 1 above)
Comment (vimal-oneway) [reply src/utils/ValidateApiKey.ts:1]: can we use axios instead of sse?

Comment (Royal-lobster) [reply src/utils/ValidateApiKey.ts]: Oh nope, i mean for verification. sse for chat is fine

**Pull request closed without merging**